From 166ccc11c71ce7d68073e4231c6423a8eae9c1c2 Mon Sep 17 00:00:00 2001
From: Karla Felix <karla.karolinenogueirafelix@windriver.com>
Date: Wed, 18 Oct 2023 19:32:16 -0300
Subject: [PATCH] Do not set gecos when creating user

Remove the gecos creation since it causes issue while running
debootstrap.

Signed-off-by: Charles Short <charles.short@windriver.com>
---
---
 debian/docker-registry.postinst | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/docker-registry.postinst b/debian/docker-registry.postinst
index d179197..d6a93ba 100755
--- a/debian/docker-registry.postinst
+++ b/debian/docker-registry.postinst
@@ -7,7 +7,6 @@ if [ "$1" = 'configure' ]; then
 			--home /var/lib/docker-registry \
 			--no-create-home \
 			--group \
-			--gecos 'Docker Registry' \
 			docker-registry
 	fi
 
-- 
2.25.1

